ansible_collections/arista/avd/roles/eos_cli_config_gen/schemas/schema_fragments/daemon_terminattr.schema.yml

```diff
@@ -54,14 +54,29 @@ keys:
               keys:
                 method:
                   type: str
-                  valid_values: ["token", "token-secure", "key"]
+                  valid_values: ["token", "token-secure", "key", "certs"]
                 key:
                   type: str
                 token_file:
                   type: str
                   description: |
                     Token file path
                     e.g. "/tmp/token"
+                cert_file:
+                  type: str
+                  description: |
+                    Client certificate file path
+                    e.g. "/persist/secure/ssl/terminattr/primary/certs/client.crt"
```

Comment:
yes but you need to know the path, in general it is `/persist/secure/ssl/terminattr/<cvoptName>/certs/client.crt`
where `cvpoptName` is always `primary` in normal configs (streaming to one cluster (single or multinode or CVaaS))
or the arbitrarily chose name by the customer when streaming to multiple CVP clusters (multiple multi-nodes, or multiple single nodes, or CVaaS tenants or a combination of all these)

Reply:
ah ok so rephrasing my question - would this work with any certs outside of this pattern:
```
/persist/secure/ssl/terminattr/<cvoptName>/certs/client.crt
```

? 

basically `/tmp/cert` -> I would assume not really given the mechanism to auto renew but maybe I am wrong :) otherwise we could add in the comment the pattern you mention above?

Reply:
No the difference with this method is that it no longer expects CV to sign any cert. It will just validate both ways towards their known CAs.